src/DayPicker.js

```diff
@@ -429,10 +468,17 @@ export default class DayPicker extends Component {
         tabIndex = this.props.tabIndex;
       }
     }
+
+    const ariaLabel = this.props.localeUtils.formatDate ?
+      this.props.localeUtils.formatDate(day) : day.toDateString();
```

Comment:
The `formatDate` here is a new function to add to the [LocaleUtils](http://www.gpbl.org/react-day-picker/docs/LocaleUtils.html) right? I can take care of this (need to update docs, tests, etc. :smile:) 
We should add the `locale` props as second argument to the function, so implementers can use their own localization utils.